@here/harp-mapview/lib/MapView.ts

```diff
@@ -180,7 +180,8 @@ const CONTEXT_LOST_EVENT: RenderEvent = { type: MapViewEventNames.ContextLost }
 const CONTEXT_RESTORED_EVENT: RenderEvent = { type: MapViewEventNames.ContextRestored } as any;
 const COPYRIGHT_CHANGED_EVENT: RenderEvent = { type: MapViewEventNames.CopyrightChanged } as any;
 
-const tmpVector = new THREE.Vector2();
+const vector2 = new THREE.Vector2();
```

Comment:
Why renaming? Also, `vector2` is quite generic and doesn't provide info about this being a temporary object.

Reply:
A `vector3` variable was also created just below, some others like `quaternion1`, `quaternion2`, `matrix1`, `matrix2`... in MapViewUtils in this PR...

Not sure that this is a worthwhile topic to debate, but here are my motivations if need be:
- "tmp" is an abbreviation, we use to avoid abbreviations, it should be "temporary",
- but "temporary" is very long,
- then it should actually be "cached", because these are _cached_, constant variables that we create to _avoid temporary objects_,
- but then I find it personally a bit of a pleonasm, their use is obviously the one of cached variables created to avoid temporary ones, when we see how they are created out of class scopes. There is also a comment telling it in all places except here, unfortunately.
- especially when one joins ordinal numbers to them.

In short I found it more readable, but I frankly don't care much. I'm open to drawbacks I didn't see.